### PR TITLE
create brayns immersive variant

### PR DESCRIPTION
**Pull request by ppodhajski**

### CODE REVIEW

**Comment by ppodhajski** [comment]
@pramodk  CI fails as this variant requires libcuda.so (/usr/lib/libcuda.so) which is available on volta partition only.

**Comment by ppodhajski** [comment]
retest this please

File: deploy/packages/bbp-packages.yaml [hunk 1]
```diff
@@ -62,6 +62,7 @@ packages:
       - touchdetector@5.0.1
       - touchdetector@5.1.0
       - brayns@0.8.0 +opendeck +brion ^ospray@1.7.3
+      - brayns@immersive +opendeck +brion +optix
```
Comment (pramodk) [inline deploy/packages/bbp-packages.yaml:65]: In addition to this, we should updated `deploy/configs/applications/modules.yaml` with:

```
      - brayns@0.8.0
      - brayns@immersive`
```

**Review by pramodk — APPROVED** [review]
Apart from modules, LGTM

**Comment by ppodhajski** [comment]
retest this please

**Comment by pramodk** [comment]
@ppodhajski : please delete the branch.

**Pull request merged**